src/main/java/com/avail/optimizer/L2ValueManifest.java

```diff
@@ -450,30 +470,54 @@ private boolean privateMergeSynonyms (
 	 * Retrieve the oldest definition of the given {@link L2SemanticValue} or an
 	 * equivalent, but having the given {@link RegisterKind}.
 	 *
+	 * @param <R>
+	 *        The kind of {@link L2Register} to return..
```

Comment:
Remove redundant period.

Reply:
Fixed, along with the others.  No other files had lines ending with "[^\.]\.\.$", so it must have been an in-file regex gone slightly awry.